plugin/execute_command.py

```diff
@@ -0,0 +1,56 @@
+import sublime
+from .core.registry import client_for_view, LspTextCommand
+from .core.settings import client_configs
+from .core.protocol import Request
+from .core.logging import debug
+from .core.rpc import Client
+
+try:
+    from typing import List, Optional, Dict, Any
+    assert List and Optional and Dict and Any
+except ImportError:
+    pass
+
+
+class LspExecuteCommand(LspTextCommand):
+    def __init__(self, view):
+        super().__init__(view)
+
+    def run(self, command_name) -> None:
+        client = client_for_view(self.view)
+        if not client:
+            self.view.show_popup("No connection with server")
+        else:
```

Comment:
When silently returning from the above `if not client:`, you can omit the `else` clause here.